File: Data Structures & Algorithms/duplicate-integer/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        // let count = 0
        const frequencies = {}
        for(let item of nums){
            frequencies[item] = (frequencies[item] || 0) + 1;
        }
        console.log(frequencies, "frequencies")
        const arr = Object.values(frequencies)
        console.log(arr, "non_arr")
        for(let count of arr){
           if(count > 1){
            return true
           }
        }
        return false

    }
}

File: Data Structures & Algorithms/majority-element/submission-1.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:

        n = len(nums)
        more = n // 2
        mp = {}

        for n in nums:
            if n in mp:
                mp[n] += 1
            else:
                mp[n] = 1
            
            if mp[n] > more:
                return n
        